src/components/Item/ItemQuantity.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+import QuantityButton from './QuantityButtons';
+import { borderRadiusMedium, colorLightGrey } from '../../styles/designTokens';
+import QuantityInput from './QuantityInput';
+
+const Wrapper = styled.div`
+  grid-column-start: 1;
+  grid-column-end: 3;
+  display: grid;
+  border-radius: ${borderRadiusMedium};
+  grid-template-columns: 3em auto 3em;
```

Comment:
Don't use em. Always use rem. It is most consistent for accessibility when user changes font size in browser etc.